src/cobra/core/gene.py

```diff
@@ -299,3 +271,360 @@ def _repr_html_(self):
             n_reactions=len(self.reactions),
             reactions=format_long_string(", ".join(r.id for r in self.reactions), 200),
         )
+
+
+class GPR(Module):
+    """A Gene Reaction rule in a cobra model, using AST as base class.
+
+    Parameters
+    ----------
+    gpr_from : Expression or Module or AST
+        A GPR in AST format
+    """
+
+    def __init__(
+        self, gpr_from: Union[Expression, Module, AST] = None, *args, **kwargs
+    ):
+        super().__init__(*args, **kwargs)
```

Comment:
I would avoid receiving `*args` in a constructor unless that is required?

```suggestion
    def __init__(
        self, gpr_from: Union[Expression, Module, AST] = None, **kwargs
    ):
        super().__init__(**kwargs)
```